src/assets/css/main.css

```diff
@@ -430,3 +430,19 @@ br.big {
 em-emoji-picker {
   width: 100%;
 }
+
+.skip-link {
+  position: absolute;
+  top: -40px;
+  left: 0;
+  background: #000;
```

Comment:
Could you please use bootstrap class names for styling where possible? You could give the skip link the following classes: "bg-dark text-light p-2 text-decoration-none position-absolute start-0 z-3" The only properties you'll still need defined on this css class are top and transition.

Reply:
The bootstrap color classes are confusing me a bit. For example `bg-dark` is giving me a light color background. `text-light` is giving me dark text. The opposite is true too. Look at my screenshots below.
![Screenshot from 2023-06-11 13-11-04](https://github.com/LemmyNet/lemmy-ui/assets/43147936/4b065726-39e3-43bd-b2c9-57ef8843c5e5)
![Screenshot from 2023-06-11 13-11-29](https://github.com/LemmyNet/lemmy-ui/assets/43147936/d70c3a24-0663-429c-b6ba-81818438df6f)